config/split_mlog_adapter.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+type splitMlogAdapter struct {
```

Comment:
It is not very clear to me why we need this adapter. Is it not possible to just inherit the parent logger and override the log level to INFO?

If not, then this should really be in the mlog package. It feels a bit weird to have a logging struct inside a config package. We can have something like the `SugarLogger` in the mlog package.

Reply:
Not entirely sure what you mean, but the interface is not compatible with mlog so we can't just use it directly. 

I think it makes sense. It's not mlog's concern this adapter with split, it's config's concern.

Reply:
Ohh sorry about that. It's nothing to do with mlog at all. It's just a logger interface for split. I'd suggest to rename the struct to `splitLogger`. 1/5.